en/peripherals/camera.md

```diff
@@ -0,0 +1,177 @@
+# Camera Trigger
+
+The camera trigger driver allows the use of the AUX ports to send out pulses in order to trigger a camera. This can be used for multiple applications including timestamping photos for aerial surveying and reconstruction, synchronising a multi-camera system or visual-inertial navigation.
+
+In addition to a pulse being sent out, a MAVLink message is published containing a sequence number (thus the current session's image sequence number) and the corresponding timestamp.
+
+## Trigger modes
+
+Four different modes are supported, controlled by the [TRIG_MODE](../advanced_config/parameter_reference.md#TRIG_MODE) parameter:
+
+Mode | Description
+--- | ---
+0 | Camera triggering is disabled.
+1 | Works like a basic intervalometer that can be enabled and disabled by using the MAVLink command `MAV_CMD_DO_TRIGGER_CONTROL`. See [command interface](camera_trigger.md#command-interface) for more details.
+2 | Switches the intervalometer constantly on.
+3 | Triggers based on distance. A shot is taken every time the set horizontal distance is exceeded. The minimum time interval between two shots is however limited by the set triggering interval.
+4 | triggers automatically when flying a survey in Mission mode.
+ 
+> **Info** If it is your first time enabling the camera trigger app, remember to reboot after changing the `TRIG_MODE` parameter.
+
+## Trigger hardware configuration
+
+You can choose which AUX pins to use for triggering using the `TRIG_PINS` parameter. The default is 56, which means that trigger is enabled on AUX 5 and AUX 6. 
```

Comment:
So presumably we have to have a note to users that depending on their firmware some AUX pins may already be used?

Can we use a MAIN pin for Pixhawk Mini? or anything else that doesn't have an IO board/AUX?

Reply:
@bkueng Do you happen to know the answer to this? Also, who are good people to talk about using Camera? I'm pretty sure I understand this, but was hoping to talk to someone who has recent experience.

Reply:
> Can we use a MAIN pin for Pixhawk Mini? or anything else that doesn't have an IO board/AUX?

That should work too. It should automatically select the MAIN pins 5 and 6 on boards w/o IO.
However, there is a limitation: the other (aux) pins can only be used for actuators/mixer outputs if TRIG_PINS is 56 (see https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L454).

>  Also, who are good people to talk about using Camera?

I'm not sure if @dogmaphobic has experience?